x-pack/plugins/ml/public/data_frame/components/group_by_list/list.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+
+import { EuiListGroup, EuiListGroupItem, EuiPanel, EuiSpacer } from '@elastic/eui';
+
+interface ListProps {
+  list: string[];
+  deleteHandler?(l: string): void;
+}
+
+export const GroupByList: React.SFC<ListProps> = ({ deleteHandler, list }) => (
+  <EuiListGroup flush={true}>
+    {list.map((l: string) => (
```

Comment:
I think a more descriptive variable name than `l` should be used here

Reply:
Changed to `fieldName` in 6b4b1f4.